### ISSUE TITLE
Mission Statement

### ISSUE THREAD

**Issue by methylDragon** [OpenSUTD/community#4]
I realise the community repo doesn't have the mission statement which can be found on the github.io.

Should we be fixing this.

**Comment by joel-huang** [comment]
Good point, I'll leave the original mission statement here up for reference: https://docs.google.com/presentation/d/1BRb055B2SfurazxRBQQu5FeWKBy1EFxxe-gPWok0tPI/edit?usp=sharing

Next up for discussion is whether that Summer 2018 mission statement is still relevant, whether to use it officially (anything in this repo should be **the authoritative source**) and whether it's understandable enough to be used on the github.io. Would love to hear any thoughts @Fishbiscuit @tlkh
